src/components/lib/dpdk/doc.md

```diff
@@ -0,0 +1,5 @@
+## DPDK
+
+### Description
+
+### Usage and Assumptions
```

Comment:
This might be a more fitting place for the documentation you placed in `doc/`. Also a brief description of what DPDK is, and why it is in cos would be good.